include/sound/sof/header.h

```diff
@@ -119,6 +119,7 @@
 #define SOF_IPC_TRACE_DMA_POSITION		SOF_CMD_TYPE(0x002)
 #define SOF_IPC_TRACE_DMA_PARAMS_EXT		SOF_CMD_TYPE(0x003)
 #define SOF_IPC_TRACE_FILTER_UPDATE		SOF_CMD_TYPE(0x004) /**< ABI3.17 */
+#define SOF_IPC_TRACE_DMA_FREE		SOF_CMD_TYPE(0x005) /**< ABI3.20 */
```

Comment:
I see a semantic difference with the regular stream IPC: this is only used when we release the trace. The HW_FREE for regular streams is called when we do a TRIGGER_STOP, but ironically we do not have an IPC to release the stream resources.

In other words, the naming here is correct, but the naming for regular IPC was not, so we either add a comment to track this difference or come up with a better wording.